angular/src/components/export.component.ts

```diff
@@ -9,28 +10,43 @@ import { EventService } from 'jslib-common/abstractions/event.service';
 import { ExportService } from 'jslib-common/abstractions/export.service';
 import { I18nService } from 'jslib-common/abstractions/i18n.service';
 import { PlatformUtilsService } from 'jslib-common/abstractions/platformUtils.service';
+import { PolicyService } from 'jslib-common/abstractions/policy.service';
 
 import { EventType } from 'jslib-common/enums/eventType';
-import { HashPurpose } from 'jslib-common/enums/hashPurpose';
+import { PolicyType } from 'jslib-common/enums/policyType';
 
 @Directive()
-export class ExportComponent {
+export class ExportComponent implements OnInit {
     @Output() onSaved = new EventEmitter();
 
     formPromise: Promise<string>;
     masterPassword: string;
     format: 'json' | 'encrypted_json' | 'csv' = 'json';
     showPassword = false;
+    disablePrivateVaultPolicyEnabled: boolean = false;
```

Comment:
Looks good, I'd only pick another name for the flag. Maybe more like the policy itself. Disable and Enable in one word is kinda odd and mixing personal vs private
disablePersonalVaultExport ?

Reply:
maybe just `exportDisabledByPolicy` doesn't really matter where it comes from

Reply:
Renamed to `disabledByPolicy` since export is implied from the component.